### PR TITLE
Add @kazie as contributor

### PR DESCRIPTION
**Pull request by Pirayya**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,9 +2,10 @@
 name = "joel-bot"
 version = "1.0.2"
 authors = [
-    "Fabian Eriksson <fabian.eriksson@r2m.se>",
+    "Fabian Eriksson <fabian.eriksson@magello.se>",
     "Joakim Anell <an3ll@hotmail.com>",
     "Simon Olander <supersimon@spray.se>",
+    "Petter Salminen <kazie@nordicfuzzcon.org>",
```
Comment (kazie) [inline Cargo.toml:8]: Ah, har jag fel git profil med fel mail. :)

Comment (kazie) [reply Cargo.toml:8]: That's a dead email now, so please use: `kemikaze+github@gmail.com`, which will be reflected in future commits.

Comment (Pirayya) [reply Cargo.toml]: Fixed!

**Review by simonolander — APPROVED** [review]
😩👌

**Pull request merged**